### ISSUE TITLE
Add disclaimer to table comparing DID methods

### ISSUE THREAD

**Issue by jceb** [decentralized-identity/did-traits#20]
The table is just an example of who DID methods can be compared based on the machine readable data.

**Comment by ottomorac** [comment]
Yes also I may suggest that we date the table as a point in time (for example January 2025 for this first version) as exemplified in this PR. https://github.com/decentralized-identity/did-traits/pull/24
